rustls/src/server/server_conn.rs

```diff
@@ -242,6 +242,30 @@ impl<'a> ClientHello<'a> {
 /// * [`ServerConfig::cert_compression_cache`]: caches the most recently used 4 compressions
 /// * [`ServerConfig::cert_decompressors`]: depends on the crate features, see [`compress::default_cert_decompressors()`].
 ///
+/// # Sharing resumption storage between `ServerConfig`s
```

Comment:
Tiny nit: I think an empty line after the heading would make this easier to scan.